catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -18,17 +18,22 @@
     <bean id="securityAttributes"
           class="org.codice.alliance.catalog.core.api.impl.types.SecurityAttributes"/>
 
+    <reference id="systemHighAttributes"
+               interface="org.codice.ddf.security.SystemHighAttributes"
+               availability="mandatory"/>
```

Comment:
I might be misunderstanding blueprint here, but I want the DefaultSecurityAttribute bean to wait to start until a SystemHighAttributes is available. I don't want to attempt to start it immediately and then fail to start because a SystemHighAttributes isn't available yet.

Reply:
As an FYI, `mandatory` is the default value for the `availability` attribute, so we technically don't have to specify it.